listenbrainz/troi/export.py

```diff
@@ -0,0 +1,23 @@
+import spotipy
+from troi.core import generate_playlist
+from troi.patches.playlist_from_listenbrainz import TransferPlaylistPatch
+
+
+def export_to_spotify(lb_token, spotify_token, playlist_mbid, is_public):
+    sp = spotipy.Spotify(auth=spotify_token)
+    # TODO: store spotify user ids in external_service_oauth table
```

Comment:
Should this be implemented in this PR or be added as a ticket?

Reply:
LB-1158